### ISSUE TITLE
What is missing for VSync, or higher FPS in general? 

### ISSUE THREAD

**Issue by MarcelHB** [gemrb/gemrb#1746]
#### Description

Unless one modifies the code, the fps is capped to 30fps. It would be nice to have an option to sync the framerate to the refresh rate of the display, and in general this raises a question: What's actually preventing us from implementing that so far?

So the benefits are for example: mouse cursor isn't slowing down (in windowed mode), scrolling a text area or flying over the map can be a lot smoother.

Problems I've observed so with some quick tests (without _any_ capping):
- Moving around the map by pushing the cursor to the borders is way too fast.
- ...

Is there anything else that I missed and that needs to be addressed first w. r. t. higher FPS? Scripts, actions, scheduling, ...?

**Comment by lynxlynxlynx** [comment]
See discussion here https://github.com/gemrb/gemrb/issues/144#issuecomment-585403875, we need to at least implement that max ini key.

**Comment by MarcelHB** [comment]
Of course, that's what I had in mind for this kind of enhancement.

**Comment by burner1024** [comment]
> just FYI, even in the originals boosting the frame rate had other consequences. generally everything runs faster and its really not a great experience. We would have to do quite a bit to make game logic and animations run from a clock rather than happen every "tick" of the game loop

I beg to differ, for me the original 30 FPS is not a great experience.

And unless I misunderstand, it's actually the other way around. I've tried GemRB with 90 fps (btw, `NOFPSLIMIT` seems to do nothing, so modified cap in `Video.h`), and the only differences I see with 30 fps version is cursor move screen speed and pendulum swing speed. I would say, twice as fast, although one would expect 3 times.

**Comment by lynxlynxlynx** [comment]
Yeah, people like to run at higher AI framerates. I don't understand the appeal, but it has been requested often enough.

AFAIK the originals had no drawing-related FPS settings, hence the confusion.

**Comment by burner1024** [comment]
It certainly would be nice to have 2 different settings, one for game speed, and another one for drawing interface.
But still, I imagine the first step would be to tie the whole game process except the interface into one timer/clock/setting.

**Comment by MarcelHB** [comment]
Sure, this topic probably cannot be addressed without having two decoupled clocks, which we then can also try to parameterize.
